File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/V3StopsByDistanceResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class V3StopsByDistanceResponse {
    /// <summary>
    /// Train stations, tram stops, bus stops, regional coach stops or Night Bus stops
    /// </summary>
    /// <value>Train stations, tram stops, bus stops, regional coach stops or Night Bus stops</value>
    [DataMember(Name="stops", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "stops")]
    public List<V3StopGeosearch> Stops { get; set; }

    /// <summary>
    /// Disruption information applicable to relevant routes or stops
    /// </summary>
    /// <value>Disruption information applicable to relevant routes or stops</value>
    [DataMember(Name="disruptions", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "disruptions")]
    public Dictionary<string, V3Disruption> Disruptions { get; set; }

    /// <summary>
    /// Gets or Sets Status
    /// </summary>
    [DataMember(Name="status", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "status")]
    public V3Status Status { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class V3StopsByDistanceResponse {\n");
      sb.Append("  Stops: ").Append(Stops).Append("\n");
      sb.Append("  Disruptions: ").Append(Disruptions).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/v3_stops_by_distance_response.dart
part of swagger.api;

class V3StopsByDistanceResponse {
  /* Train stations, tram stops, bus stops, regional coach stops or Night Bus stops */
  List<V3StopGeosearch> stops = [];
/* Disruption information applicable to relevant routes or stops */
  Map<String, V3Disruption> disruptions = {};

  V3Status status = null;

  V3StopsByDistanceResponse();

  @override
  String toString() {
    return 'V3StopsByDistanceResponse[stops=$stops, disruptions=$disruptions, status=$status, ]';
  }

  V3StopsByDistanceResponse.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    stops = V3StopGeosearch.listFromJson(json['stops']);
    disruptions = V3Disruption.mapFromJson(json['disruptions']);
    status = new V3Status.fromJson(json['status']);
  }

  Map<String, dynamic> toJson() {
    return {
      'stops': stops,
      'disruptions': disruptions,
      'status': status
     };
  }

  static List<V3StopsByDistanceResponse> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3StopsByDistanceResponse>() : json.map((value) => new V3StopsByDistanceResponse.fromJson(value)).toList();
  }

  static Map<String, V3StopsByDistanceResponse> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3StopsByDistanceResponse>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3StopsByDistanceResponse.fromJson(value));
    }
    return map;
  }
}
